src/index.js

```diff
@@ -1,14 +1,65 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+let newTask;
+const arrowBtn = document.getElementById('arrow');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+class Todolist {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
 
-  return element;
+  addtask(dataBase) {
+    dataBase.push(this);
+  }
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+arrowBtn.addEventListener('click', () => {
+  const taskName = document.getElementById('input').value;
+  let dataBase = JSON.parse(localStorage.getItem('baseData'));
+  if (dataBase === null) {
+    dataBase = [];
+  }
+  const index = dataBase.length + 1;
+  newTask = new Todolist(taskName, false, index);
+  newTask.addtask(dataBase);
+  localStorage.setItem('baseData', JSON.stringify(dataBase));
+  document.location.reload();
```

Comment:
- [ ] Your addition button is working. But, you are reloading the page after adding it to the localStorage, Please create a render method that should render the list of todos after adding it. method to render your new todo in the list instead of reloading the whole document :+1:

Reply:
fullfield 👍🏾